include/zephyr/dt-bindings/clock/ch32v00x-clocks.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 Michael Hope
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_GD32_COMMON_H_
```

Comment:
This guard not seems right. I can see that many files were copied and reworked but all the copyrights were removed.